pagerduty/resource_pagerduty_user.go

```diff
@@ -44,6 +44,17 @@ func resourcePagerDutyUser() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 				Default:  "user",
+				ValidateFunc: validateValueFunc([]string{
+					"admin",
+					"limited_user",
+					"observer",
+					"owner",
+					"read_only_user",
+					"restricted_access",
+					"read_only_limited_user",
+					"user",
+					"team_responder",
```

Comment:
Looks like the 'team_responder' role has been deprecated: https://community.pagerduty.com/forum/t/creating-user-via-api-with-role-team-responder-sets-observer/1794
https://gist.github.com/fussybeaver/4ba9a98fed7c0c9c836a04ade3a108f3#file-json-L1836

Looks good otherwise.

Reply:
removed it.